docs/software/system_stats.md

```diff
@@ -417,4 +418,51 @@ Official documentation: <https://rancher.com/docs/k3s/latest/en/>
 Source code: <https://github.com/k3s-io/k3s>  
 License: [Apache 2.0](https://github.com/k3s-io/k3s/blob/master/LICENSE)
 
+## Microk8s
+
+High availability - Low-ops, minimal production Kubernetes, for devs, cloud, clusters, workstations, Edge and IoT.
+
+![microk8s logo](../assets/images/microk8s.png){: width="300" height="116" loading="lazy"}
+
+
+
+=== "Connecting to your cluster"
+
+    To create a cluster out of two or more already-running MicroK8s instances, use the microk8s add-node command
+
+    ```sh
+    microk8s add-node
+    From the node you wish to join to this cluster, run the following:
```

Comment:
Within a shell highlight code block, either make such a comment or separate it into three dedicated single-line blocks with the explanation as Markdown text between them.